tests/migrations/0001_initial.py

```diff
@@ -31,6 +31,21 @@ class Migration(migrations.Migration):
                 ("title", models.CharField(max_length=100)),
             ],
         ),
+        migrations.CreateModel(
```

Comment:
did you created the migrations yourselves? are they going to break the downward migrations?

Reply:
> did you created the migrations yourselves?

Yes.

> are they going to break the downward migrations?

Please review them. The tests are passing.